torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -238,7 +238,10 @@ def fn(match, *args, **kwargs):
             ):
                 matched = False
             else:  # inp is a Number
-                matched = True
+                if min_value > max_value:
```

Comment:
Please use a bool value assignment instead of control flow here.

Reply:
yes, changed.